application/common/mail/welcome.text.php

```diff
@@ -25,57 +25,21 @@
 ?>
 Dear <?= $displayName ?>,
 
-Congratulations! You have logged into your new <?=$idpDisplayName . PHP_EOL ?>
-account for the first time. Please remember that this account will be your
-primary means for logging into many corporate applications. It is also
-important to note that this account (username and password) is unique and
-will not be kept in sync with any other accounts you have.
+<p>
```

Comment:
Since this is the plain text email we need to remove all the html tags and just use whitespace for formatting.